doc/user_guide/Parameters.ipynb

```diff
@@ -784,6 +784,156 @@
+  },
+  {
+   "cell_type": "markdown",
+   "id": "324475cf-65a3-4902-8508-57af19697b5f",
+   "metadata": {},
+   "source": [
+    "By declaring that `V.b` allows references we have made it possible to pass the Parameter `U.b`, which means `v.b` will reflect the value of `u.a`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "62016252-5a7d-4bb7-be7a-029ed1a47025",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "u.a = 1.57\n",
+    "\n",
+    "v.b"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c6f62936-89bb-49e3-b197-cef43891b2f7",
+   "metadata": {},
+   "source": [
+    "This unidirectional link will be in effect until something else tries to set the value:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "28094868-4f23-4399-be7b-da0c73854be7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "v.b = 14.1\n",
+    "u.a = 13.2\n",
+    "\n",
+    "v.b"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48878238-e062-4fd4-bb99-0e9020d9b26f",
+   "metadata": {},
+   "source": [
+    "In other words, if the value is overridden from the outside the link will be automatically removed.\n",
+    "\n",
+    "Simple references are resolved when `allow_refs=True` but to allow nested references we separately have to set `nested_refs=True`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d63d949d-79a2-471f-b0bb-232086e486bf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class W(V):\n",
+    "    \n",
+    "    c = param.List(allow_refs=True, nested_refs=True)\n",
+    "    \n",
+    "u1 = U(a=3)\n",
+    "u2 = U(a=13)\n",
+    "\n",
+    "w = W(c=[u1.param.a, u2.param.a]) \n",
+    "\n",
+    "w.c"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e7b3489f-4a75-4767-a7d2-b7066a976ce4",
+   "metadata": {},
+   "source": [
+    "When we modify either `u1.a` or `u2.a`, `w.c` will update:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c2d9fc30-d5cb-49ac-aaf7-d4d6f1d74cb0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "u1.a = 7\n",
+    "\n",
+    "w.c"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae2d21b3-c78b-4648-8622-282c237a1d64",
+   "metadata": {},
+   "source": [
+    "Note that `Parameter` types are not the only types of valid references. The full list of valid references include:\n",
+    "\n",
+    "- Class and instance `Parameter` objects\n",
+    "- Functions or methods annotated with `param.depends`\n",
+    "- Functions wrapped with `param.bind`\n",
+    "- Reactive expressions declared using `param.rx`\n",
+    "- Asynchronous generators\n",
+    "- Custom objects transformed into a valid reference with a hook registered with `param.parameterized.register_reference_transform`.\n",
+    "\n",
+    "There are two utility functions which allow resolving the value of a reference and all parameters the reference depends on:"
```

Comment:
The order in the sentence (value/reference) is inverted compared to the order of the example below.